core/idl/wallet/common/wallet.djinni

```diff
@@ -348,6 +357,20 @@ Account = interface +c {
     const EV_NEW_OP_WALLET_NAME: string = "EV_NEW_OP_WALLET_NAME";
     const EV_NEW_OP_ACCOUNT_INDEX: string = "EV_NEW_OP_ACCOUNT_INDEX";
     const EV_NEW_OP_UID: string = "EV_NEW_OP_UID";
+
+    # Returned flag when a transaction is put in DB
+    # Some of those are unrelevant for specific coins
+    # but for the sake of generic mechanism we put all those
+    # flags together
+    const FLAG_TRANSACTION_IGNORED: i32 = 0;
```

Comment:
When this is used ? when there is no new transaction to retrieve ?

Reply:
It's the default value used by all coins when we try to put a transaction on an account

Reply:
I cannot certify that the meaning is the same for all the coins